middlewares/authnToken.js

```diff
@@ -46,7 +46,7 @@ module.exports = (req, res, next) => {
                 token_id: result.rows[0].token_id,
             };
             sqldb.query(sql.update_token_last_used, lastUsedParams, (err) => {
-                if (ERR(err, (e) => logger.error(e)));
+                if (ERR(err, (e) => logger.error('Error in update_token_last_used', e)));
```

Comment:
```suggestion
                if (ERR(err, (e) => logger.error('Error in sql.update_token_last_used', e)));
```
Originally suggested `query()`, but seems like this deviation was intended since I saw it again below. Maybe add `sql.`?